### DISCUSSION TITLE
Facing a problem : initializing miniconda3

### DISCUSSION THREAD

**Discussion by yangso1** [mhm-ufz/mHM#55] [Installation] [answered]
Dear mHM-developers,

I need your help to activate 'Miniconda3 initialization' in my Linux system.
In fact, by following the mHM installation steps, I succeed to install the Miniconda3 into my system. However, I said "no" for the question on " Do you wish the installer to initialized Miniconda3 by running conda init?". 

Since I followed the manual statement in sequence from top, I thought that the question indicates the need to run an un-related program additionally. That's why I said "no"...

However, I realized that it must be activated to get done for mHM installation. Could somebody help me to activate it?
![Picture1](https://user-images.githubusercontent.com/107365830/173247514-1724e265-5e04-498d-8ddb-8340c4b02a8a.png)

Here I copied the screenshot of my Linux installer.

Best,

**Comment by MuellerSeb** [comment, marked as answer]
Hi there.

You could either fully remove the installed miniconda folder and do it again with answering this question with `yes`, or you could run `conda init` manually and then restart your bash:
```bash
./miniconda3/bin/conda init
bash
```
Since `conda init` was not run, the `conda` command is not yet available since it was not added to your `PATH`. That is why you need to provide the full path to the conda binary (`./miniconda3/bin/conda` in this case, please replace with another path, if you have installed miniconda somewhere else).

Hope that helps.
Sebastian
